### ISSUE TITLE
send Response Function

### ISSUE THREAD

**Issue by dhershman1** [dhershman1/octoris#6]
`send` is a function for the [Response](https://github.com/dhershman1/octoris/issues/3) module of octoris. It's prime use is to just send data back to the client

## Concept

We set status on the response

```js
const { send } = require('octoris/response')

// Works
send(200, 'You did a thing!')

// Does not work still
send(200, { a: 1 })
```

This approach is nice because we tell our response "everything is 200 OK here is some data" so it's more straight forward and easy to see what is going on.

**Comment by Glen-Cocoa** [comment]
Status should be an optional param, IMO, with the default being 200

I like proposal 1, though more clarity in the function name would be beneficial, i.e. instead of `setStatus` 
something like `setResponseStatus` or `setResponseHttpStatus` (though that last one might be a bit wordy)

edit: On second thought and when looking at issue #4, it is more intuitive to be able to just
`send(200, {content})` as is done in proposal 2

**Comment by dhershman1** [comment]
I am going with breaking down the functionality.

- `send` is for normal text/html content types `send(200, 'hello thing')`
- `json` is for json object/array based data `json(200, { hello: 'thing' })`

Proposal 1 is a for sure negative, a magic function that mutates the context in order to create a status.

Proposal 2 is going to be the best bet in terms of how the syntax and functionality layout will work.

**Comment by dhershman1** [comment]
Closing this for now since send is added and working correctly within the response API
